terraform/locals.tf

```diff
@@ -7,4 +7,8 @@ locals {
     zimbatm    = "zimbatm@zimbatm.com"
     zowoq      = "zowoq.gh@gmail.com"
   }
+
+  bot = {
+    nix-infra-bot = "admin@nix-community.org"
```

Comment:
@Mic92 Can we transfer this account so it is controlled by the org and we store the login in sops? Could create another github account if that is easier?

Reply:
done.

Reply:
Thanks, I'll reset the password, add 2fa and add it to secrets.yaml in the repo root.